kafka/librdkafka/rdkafka.h

```diff
@@ -55,13 +55,16 @@ extern "C" {
 #endif
 #endif
 
-#ifdef _MSC_VER
+#ifdef _WIN32
```

Comment:
i assume this file is copied direct from librdkafka.

Reply:
Yep